.github/workflows/acceptance.yml

```diff
@@ -122,7 +115,7 @@ jobs:
       # it reduces large coverage fluctuations
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
-        if: ${{ always() && needs.files-changed.outputs.frontend_all == 'true' }}
+        if: always() && needs.files-changed.outputs.frontend_all == 'true'
```

Comment:
Okay. This does not work. Let's go back.
<img width="195" alt="image" src="https://user-images.githubusercontent.com/44410/214311959-6eaefe25-5bce-4a31-9db5-15ac66b29373.png">